tests/e2e/sections/onboarding/BusinessSection.ts

```diff
@@ -46,7 +46,7 @@ export class BusinessSection extends BasePage {
 			const inputsNum = document.querySelectorAll(
 				'.components-checkbox-control__input'
 			).length;
-			return inputsNum > 6;
+			return inputsNum > 5;
```

Comment:
This is counting extensions from WCCOM right?  It seems odd we're validating data outside the repo and prone to break again in the future.

Could we safely check for the presence of any (`>= 1`) option?

Reply:
I'm not sure what the source of the extensions is tbh.. I think a `> 1` check would be safe enough - the select all plus one or more extension recommendation.

Reply:
Done in b420cfe.